components/raftstore/src/store/peer.rs

```diff
@@ -1943,11 +1925,11 @@ where
                 committed_entries,
                 cbs,
             );
+            self.mut_store().trace_cached_entries(apply.entries.clone());
```

Comment:
Why do we need to change it?

Reply:
`trace_cached_entries` must be called before `evict_cache`. Otherwise `evict` can't release dangle entries (entries not in the entry cache).